src/parsers/js/transformers/babel6/index.js

```diff
@@ -26,7 +26,7 @@ export default {
     );
 
     return babel.transform(code, {
-      presets,
+      presets: [],
```

Comment:
Interesting.. I kept wondering why I couldn't use any stage-0 features in the code and got unexpected token.

It's because of `halts(transformCode);` which uses acorn and doesn't understand stage-x syntax.